presto-main/src/main/java/com/facebook/presto/execution/QueryManagerStats.java

```diff
@@ -75,7 +76,6 @@ private void queryStarted()
     {
         startedQueries.update(1);
         runningQueries.incrementAndGet();
-        queryDequeued();
     }
 
     private void queryStopped()
```

Comment:
I think we should introduce `queryQueued()` method, similar to these methods.

Reply:
Done.